website/docs/zh-CN/input.md

```diff
@@ -951,7 +951,7 @@ export default defineComponent ({
 |-------------  |---------------- |---------------- |---------------------- |-------- |
 | type         | 类型   | string  | text，textarea 和其他 [原生 input 的 type 值](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#Form_%3Cinput%3E_types) | text |
 | model-value / v-model | 绑定值           | string / number  | — | — |
-| maxlength     | 原生属性，最大输入长度      | number          |  —  | — |
+| maxlength     | 最大输入长度      | number          |  —  | — |
```

Comment:
Reminder:
You need to update all language documentations, if the documentation’s language is not your familiar language, please use translator to translate it direct to English, for all languages.